src/imitation/scripts/analyze.py

```diff
@@ -123,7 +126,13 @@ def gather_tb_directories() -> dict:
                 os.makedirs(symlinks_dir, exist_ok=True)
 
                 tb_symlink = osp.join(symlinks_dir, run_name)
-                os.symlink(tb_src_dir, tb_symlink)
+                try:
+                    os.symlink(tb_src_dir, tb_symlink)
+                except OSError as e:
+                    if os.name == "nt":  # Windows
+                        print("Exception occurred while creating symlink.")
+                        print("Please ensure that Developer model is enabled.")
+                    raise e
```

Comment:
print could be easily be missed (goes to stdout not stderr, some execution environments will catch exception and log them but not stdout/stderr, etc). Would favour adding this message directly to exception. See https://stackoverflow.com/questions/9157210/how-do-i-raise-the-same-exception-with-a-custom-message-in-python for methods. Simplest is something like:

```raise OSError("Exception occurred while creating symlink ....") from e```